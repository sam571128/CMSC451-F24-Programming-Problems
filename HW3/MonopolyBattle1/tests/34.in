30 900 1000
30 23
23 14
20 3
19 16
13 2
14 16
3 12
20 15
24 29
17 24
6 12
29 27
13 13
25 6
3 13
12 22
2 16
27 9
5 6
12 12
25 21
29 22
21 22
18 12
8 3
28 7
23 29
5 2
21 4
21 23
19 19
14 26
29 28
29 16
18 16
22 14
5 17
10 12
3 7
13 27
1 4
9 22
27 20
28 1
23 8
27 11
12 8
18 1
4 26
24 8
30 6
4 29
6 16
8 15
14 14
24 16
22 5
20 12
4 8
10 21
28 15
2 9
26 5
17 22
30 3
12 16
8 23
16 2
18 29
21 5
11 28
8 24
1 14
26 25
5 23
30 27
18 20
27 17
28 29
12 11
29 18
30 12
16 29
25 25
16 12
14 20
14 28
3 6
15 19
15 8
10 3
5 30
17 13
3 21
23 24
15 13
17 4
29 3
12 7
27 14
21 2
1 20
6 8
20 5
5 25
9 27
19 27
5 10
19 29
21 27
26 10
30 11
11 29
20 25
25 3
28 25
26 2
10 14
11 11
2 20
4 30
15 14
19 18
23 9
15 12
19 13
1 3
12 29
10 28
3 20
5 3
22 20
28 30
20 21
25 28
21 11
11 2
15 23
12 3
13 5
15 15
21 13
7 7
22 12
26 24
11 18
24 4
13 8
1 15
1 24
28 5
26 7
15 16
19 8
10 10
24 25
28 4
17 2
22 7
19 9
25 15
10 22
8 25
2 6
15 26
30 13
14 18
23 26
1 23
9 23
6 14
21 1
12 18
12 10
27 10
6 1
6 23
18 24
27 28
4 17
30 25
18 4
25 24
20 18
13 18
9 19
27 4
12 25
17 10
13 19
3 24
15 4
8 4
16 4
18 7
15 25
28 23
16 13
12 26
5 15
4 10
2 26
28 19
18 27
28 18
28 13
25 7
24 5
16 16
19 1
10 20
2 7
16 1
10 1
26 11
21 20
2 2
2 25
13 23
2 24
27 26
16 23
2 5
24 15
16 10
15 7
22 4
4 25
10 19
25 1
24 22
14 2
11 15
30 28
24 12
28 26
14 5
10 5
30 29
4 13
24 7
17 15
13 14
13 6
28 28
12 13
19 26
25 5
3 10
8 8
8 27
26 21
29 14
29 4
16 21
27 15
7 11
24 21
12 2
16 11
11 9
25 16
20 11
12 14
29 23
17 14
3 15
22 17
24 13
21 7
10 16
22 30
25 12
22 27
18 22
1 25
10 25
12 4
20 27
30 2
27 7
12 1
27 2
2 30
25 9
14 8
26 9
16 14
19 20
12 9
28 24
29 20
19 3
26 13
16 26
11 3
9 1
23 27
19 12
2 27
16 18
15 21
9 30
23 16
19 24
26 17
9 26
23 25
30 8
3 18
11 13
13 26
27 8
16 7
21 6
17 16
2 8
11 12
9 25
4 21
15 30
9 24
7 23
8 13
21 14
8 26
3 19
26 6
20 20
7 1
27 13
29 21
5 19
2 28
15 9
5 26
10 8
14 7
18 19
7 26
2 29
18 14
13 4
29 15
25 29
7 5
1 12
21 25
25 14
16 30
26 14
25 22
21 3
12 21
9 2
16 27
10 13
21 8
30 10
30 5
12 30
24 17
28 17
23 15
2 23
26 22
15 6
14 6
18 21
12 15
18 13
3 27
6 20
14 23
11 19
17 8
2 4
5 5
6 24
28 21
2 13
30 14
28 3
21 28
30 20
3 30
11 22
14 11
10 27
8 21
22 23
13 9
15 29
15 5
14 13
16 6
6 15
25 30
22 21
16 22
1 30
14 4
18 26
23 18
17 25
1 11
25 26
4 6
14 15
24 2
29 13
20 28
22 22
30 30
1 1
21 18
24 19
25 20
7 21
6 6
3 26
18 18
10 17
14 9
4 5
20 7
29 12
10 23
26 30
4 1
4 12
27 27
15 22
12 28
30 18
8 14
18 9
4 18
26 19
11 21
18 3
5 16
1 27
29 9
24 27
26 4
21 12
9 18
11 8
13 1
2 14
2 3
30 16
9 16
21 21
23 22
27 30
7 4
15 20
26 15
24 1
11 10
2 22
16 8
30 19
1 5
17 26
12 20
11 6
9 13
29 5
11 14
29 10
8 19
22 18
23 6
8 28
7 15
22 19
2 19
29 26
9 15
4 27
21 29
14 17
12 5
6 27
7 18
24 11
19 7
22 25
29 30
26 26
1 6
24 26
11 17
18 25
16 3
28 27
26 16
2 15
14 10
20 19
9 3
7 12
23 11
9 6
22 6
6 17
4 9
14 30
16 15
22 8
6 7
19 30
14 22
22 13
19 2
7 6
20 24
29 1
24 3
12 27
30 7
23 23
23 20
16 28
15 18
17 11
7 13
9 4
18 8
6 10
4 20
28 2
25 17
10 6
9 8
13 22
6 26
11 7
8 10
4 2
28 22
27 6
25 4
3 23
17 6
23 21
17 12
15 17
19 17
1 28
17 5
19 25
9 5
13 28
8 17
5 21
29 8
9 12
20 10
1 8
17 23
18 23
27 22
11 25
26 18
8 16
13 12
22 11
2 11
26 12
11 30
13 11
17 30
21 15
23 13
7 8
3 11
17 20
8 1
5 13
12 23
17 7
8 29
21 10
5 22
3 16
14 27
3 1
10 2
2 21
29 25
25 10
15 3
13 20
8 18
6 9
20 30
12 17
3 2
27 3
3 8
2 10
8 22
28 12
3 17
3 29
9 11
1 10
11 16
4 15
11 20
1 13
15 24
5 18
24 20
19 23
13 16
24 6
19 14
23 17
10 4
25 8
13 24
28 6
23 3
3 28
24 9
29 7
22 29
15 1
19 5
12 24
29 2
24 30
25 23
27 29
9 20
16 24
3 3
25 11
2 18
13 21
20 9
6 22
24 23
6 25
9 7
21 26
8 9
3 4
17 19
14 19
20 23
1 26
13 30
7 27
8 7
30 15
6 21
27 1
3 5
19 11
9 21
30 22
27 16
30 21
26 1
26 3
29 17
22 10
18 2
16 5
18 5
18 30
4 28
6 30
29 29
23 7
21 16
7 3
4 23
10 24
2 12
27 18
7 16
4 24
14 29
18 6
6 3
24 18
15 28
5 28
22 1
5 12
1 7
26 20
3 25
18 10
29 24
10 18
4 14
25 13
5 27
22 9
18 11
24 10
1 18
19 6
7 9
13 29
4 3
21 24
20 1
9 9
5 9
28 10
11 24
11 23
17 27
22 24
8 2
17 18
4 22
23 12
13 10
13 3
20 2
13 25
9 14
27 5
24 28
4 16
5 11
25 2
25 27
8 11
22 15
29 19
6 2
5 20
28 16
11 5
17 17
14 3
23 5
26 23
24 14
10 11
30 17
16 19
30 4
23 4
1 2
11 1
8 30
1 29
14 25
21 19
15 11
23 2
8 6
1 19
30 9
13 7
5 4
6 11
7 29
9 29
6 18
6 13
10 29
1 16
23 1
7 14
12 6
19 21
3 9
16 9
4 19
20 13
21 9
10 30
4 7
23 10
14 12
20 22
18 15
23 28
20 26
20 8
22 2
12 19
27 12
17 9
6 4
4 4
26 27
9 10
18 28
6 19
19 15
7 20
1 22
30 26
6 29
14 21
10 15
13 17
3 14
1 9
19 10
8 5
8 20
27 21
28 11
29 11
3 22
16 20
10 9
28 20
7 22
26 29
30 24
19 4
24 24
5 1
5 7
17 1
10 26
7 30
15 10
22 16
19 22
27 24
14 24
23 19
1 21
28 9
25 19
29 6
7 25
23 30
17 29
25 18
17 3
5 8
17 28
5 14
20 16
26 8
27 25
27 19
15 27
1 17
20 14
10 7
22 26
4 11
26 28
28 8
7 2
17 21
7 10
15 2
21 30
13 15
28 14
5 29
20 29
11 4
19 28
6 28
2 1
14 1
20 17
16 25
8 12
30 1
7 17
9 28
7 28
11 26
16 17
20 6
2 17
18 17
7 19
6 5
27 23
7 24
11 27
20 4
5 24
21 17
9 17
22 28
22 3
