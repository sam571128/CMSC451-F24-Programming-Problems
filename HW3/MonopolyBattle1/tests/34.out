6788 7142 7606 7142 7575 7143 7142 7142 7142 7142 7142 7142 7142 7142 7367 7142 7142 7142 7142 7142 7142 7142 7142 7142 7915 7142 7142 7737 7142 7142 